todo_app/templates/index.html

```diff
@@ -11,8 +11,15 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items%}
+            <li>{{item.title}}</li>
+            {% endfor %}
       </ul>
     </div>
   </div>
+
+  <form action="{{'/add_item'}}" method="post">
```

Comment:
No need to use `{{ ... }}` for a hardcoded constant - it can just be `action="/add-item"`

You use `{{` if you want to evaluate some code. E.g. `{{url_for('add')}}`